src/driver/mysql/MysqlQueryRunner.ts

```diff
@@ -1688,8 +1704,8 @@ export class MysqlQueryRunner extends BaseQueryRunner implements QueryRunner {
             c += " PRIMARY KEY";
         if (column.isGenerated && column.generationStrategy === "increment") // don't use skipPrimary here since updates can update already exist primary without auto inc.
             c += " AUTO_INCREMENT";
-        if (column.comment)
-            c += ` COMMENT '${column.comment.replace("'", "''")}'`;
+        if (column.comment != undefined && column.comment.length > 0)
```

Comment:
`!=` ?

Reply:
I was doing this to prevent `NULL` as well as `undefined` - holdover ideas from JavaScript.  Not needed because the type shouldn't allow null.